### ISSUE TITLE
Simplification of inverses

### ISSUE THREAD

**Issue by michakraus** [SciML/ModelingToolkit.jl#615]
I observed a minor issue when simplifying inverses:

```julia
julia> @variables x
(x,)

julia> 1 / x
1 / x

julia> simplify(1 / x)
inv(x) ^ 1

julia> inv(x)
inv(x)

julia> simplify(inv(x))
inv(x)
```
If I write `1 / x` there is a power of 1 that does not get simplified.

**Comment by ChrisRackauckas** [comment]
Ha, that's peculiar. Maybe @shashi  knows why it would do that.

**Comment by michakraus** [comment]
I found another one:
```julia
julia> simplify(sqrt(Constant(1)))
sqrt(1)
```

**Comment by isaacsas** [comment]
The `inv` and `^(-1)` issue has been ongoing, see https://github.com/SciML/ModelingToolkit.jl/issues/558. @shashi and I discussed it earlier in the week so I think it is on the radar to get looked at.

**Comment by shashi** [comment]
Yeah `/` is definitely a weird thing when it comes to simplification. I'm also not sure why it introduces `inv`.... My plan is:

- make sure `inv` does not get introduced, only negative powers do.
- simplify using exponent arithmetic
- put back `/` after simplification so that there are no negative powers.
- have an optional argument which simplifies to a rational polynomial normal form i.e. (p / q) where `p` and `q` are "polynomials" in variables and non-linear functions of them.

**Comment by shashi** [comment]
https://github.com/JuliaSymbolics/SymbolicUtils.jl/issues/133

**Comment by YingboMa** [comment]
This should be fixed by #720

**Comment by michakraus** [comment]
Thanks, works!
